ui/src/SpaceDashboard/TransferOwnershipForm.tsx

```diff
@@ -77,7 +75,7 @@ function TransferOwnershipForm({currentSpace, currentUser, closeModal, setCurren
             onClick={(): void => setSelectedUser(person)}>
             <i className={'material-icons'} aria-hidden>account_circle</i>
             <span className={'personRadioUserId'}>{person.userId.toLowerCase()}</span>
-            <input type={'radio'} name={'newOwner'} value={person.userId} checked={selectedUser ? selectedUser.id === person.id : false}/>
+            <input type={'radio'} name={'newOwner'} value={person.userId} checked={selectedUser ? selectedUser.id === person.id : false} onChange={() => {setSelectedUser(person)}}/>
```

Comment:
Were you able to test the functionality of this to ensure it didn't cause any issues in the UI? This whole thing needs to be updated cuz a div shouldn't have an onclick on it anyway. It needs to be a button or otherwise refactored, which I had on my list of todos later.

Reply:
It is covered (somewhat enough) in the unit tests. I like your suggestion to turn the div into a labal. Also, I like the idea of removing this from your todo list. :)